features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/Scenario.kt

```diff
@@ -29,26 +30,33 @@ abstract class Scenario(
     protected val eventMetadata: String?
 ) : Application.ActivityLifecycleCallbacks {
 
+    /**
+     * Tracks whether the scenario is starting Bugsnag only, or running the scenario.
+     */
+    protected var startBugsnagOnly: Boolean = false
+
     /**
      * Determines what log messages should be intercepted from Bugsnag and sent to Mazerunner
      * using a HTTP requests to the /logs endpoint. This is used to assert that Bugsnag is
      * behaving correctly in situations where sending an error/session payload is not
      * possible.
      */
-    open fun getInterceptedLogMessages() = emptyList<String>()
+    open fun getInterceptedLogMessages() = getZeroEventsLogMessages(startBugsnagOnly)
```

Comment:
The default for this should be an empty list, because otherwise mazerunner will send a HTTP request for _every_ scenario that matches these log messages. It should be overridden in the appropriate scenarios instead to minimize HTTP traffic.

Reply:
Done.